### PR TITLE
coverage: use source in testing

### PR DESCRIPTION
**Pull request by yorkie**
This patch did use real source files copied from `app` and `test` directories, rather than the generated file `public/index.js`, such that i can of course get normal and complete coverage page like the following image.
screen shot 2015-01-06 at 10 12 12 pm

@kenhkan could you help me with reviewing this PR, i'm still unsure my changes is good to Haskell, haha thanks :)

btw, do we need to make the coverage be optional, that's because the coverage process actually takes a bit long time

### CODE REVIEW

**Comment by yorkie** [comment]
@kenhkan i added summary display by 2334018, we can get a simple coverage summary now like the following:

```bash
>> Coverage Summary (74%)

   public/app/modules/analytics.js: 40%
   public/app/modules/api.js: 75%
   public/app/modules/aws/index.js: 89%
   public/app/modules/aws/kinesis.js: 35%
   public/app/modules/aws/s3.js: 95%
   public/app/modules/files.js: 100%
   public/app/modules/index.js: 4%
   public/app/modules/local-storage.js: 84%
   public/app/modules/network.js: 69%
   public/app/modules/online.js: 84%
   public/app/modules/request.js: 75%
   public/app/modules/utils-sha256.js: 82%
   public/app/modules/utils.js: 83%
   public/app/modules/view.js: 75%
```
Now feel free to review and merge this PR, thanks :)

**Comment by yorkie** [comment]
also fixed `bind` function and `uglifyjs` issues at 2251723 and 72e6f80

File: src/CLI.hs [hunk 1]
```diff
@@ -139,6 +139,9 @@ main = do
 
   -- We may need custom builds with mode
   let depManifestPath = cwd </> "component.json"
+
+  -- TODO(Yorkie): to be remove if verified by @Kenneth
```
Comment (kenhkan) [inline src/CLI.hs]: What do you mean? It's needed in `Scripts.hs`, `Static.hs`, and `Styles.hs`.

Comment (yorkie) [inline src/CLI.hs]: Okay, will remove this TODO, actually i have used the dynamic dependencies in `Script.hs`

**Pull request merged**